force_bdss/mco/optimizers/nevergrad_optimizers.py

```diff
@@ -0,0 +1,139 @@
+from traits.api import (
```

Comment:
This code should be in the `force-bdss-plugin-nevergrad` repo. Could you please make a new PR for it there?

Reply:
The PR is there (#17) but won't build because it has bdss dependencies that aren't in the master.